packages/client/src/i18n/messages/views/formConfig.ts

```diff
@@ -129,15 +131,16 @@ const navigationMessagesToDefine: INavigationMessages = {
     id: 'config.form.navigation.spouse',
     defaultMessage: 'Spouse details',
     description: 'Label for spouse details in page navigation'
-  },
-  registration: {
-    id: 'config.form.navigation.registration',
-    defaultMessage: 'Registration',
-    description: 'Label for registration in page navigation'
   }
 }
 
 const messagesToDefine: IFormConfigMessages = {
+  globalSettingsDescription: {
+    id: 'config.form.settings.globalSettingsDescription',
+    defaultMessage:
+      'These are global form settings that can be configured.  Please note that applying any changes to these settings takes effect immediately to the currently active forms independently to your draft status.',
+    description: 'Label for global form settings introduction'
```

Comment:
We need to inform the user that changing these settings has an immediate impact as we are updating application config not form config